test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
+    os::cmd::expect_success "oc cluster up --service-catalog --version=latest"
+    os::test::extended::clusterup::verify_router_and_registry
+    os::test::extended::clusterup::verify_image_streams
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success "oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated"
+    #oc::cmd::expect_success "oc adm policy add-role-to-user view developer -n openshift"
+    os::cmd::expect_success "oc login -u developer"
+    os::cmd::expect_success "pushd ${OS_ROOT}/pkg/template/servicebroker/test-scripts; serviceUUID=`oc get template jenkins-ephemeral -n openshift -o template --template '{{.metadata.uid}}'` ./provision.sh"
```

Comment:
I was referring to the service hostname, not route hostname.

That said, it appears depending on your dns configuration that may or may not be resolvable from your host machine (basically whether your host DNS setup includes the cluster dns).